src/components/CertificateVerifyBlock/CertificateVerifyBlock.js

```diff
@@ -1,129 +1,53 @@
+import React, { useState } from "react";
 import PropTypes from "prop-types";
-import { get, sortBy } from "lodash";
+import { get } from "lodash";
 import DetailedCertificateVerifyBlock from "./DetailedCertificateVerifyBlock";
-import { LOG_LEVEL } from "./constants";
 import css from "./certificateVerifyBlock.scss";
 import icons from "../ViewerPageImages";
 
-const statusSummary = ({
-  verifying,
-  hashStatus,
-  issuedStatus,
-  notRevokedStatus,
-  issuerIdentityStatus
-}) => {
-  if (verifying) return LOG_LEVEL.VERIFYING;
-  if (
-    !verifying &&
-    hashStatus.verified &&
-    issuedStatus.verified &&
-    notRevokedStatus.verified &&
-    issuerIdentityStatus.verified
-  )
-    return LOG_LEVEL.VALID;
-  if (
-    !verifying &&
-    hashStatus.verified &&
-    issuedStatus.verified &&
-    notRevokedStatus.verified &&
-    !issuerIdentityStatus.verified
-  )
-    return LOG_LEVEL.WARNING;
-  return LOG_LEVEL.INVALID;
-};
+const renderIcon = () => (
+  <div
+    className={`d-flex justify-content-center align-items-center ${
+      css["verified-icon"]
+    }`}
+  >
+    {icons.checkCircle()}
+  </div>
+);
 
-const renderIcon = status => {
-  let icon;
-  switch (status) {
-    case LOG_LEVEL.CONNECTING:
-    case LOG_LEVEL.VERIFYING:
-      icon = <i id="verify-spinner" className="fa fa-spinner fa-spin fa-2x" />;
-      break;
-    case LOG_LEVEL.VALID:
-      icon = icons.checkCircle();
-      break;
-    case LOG_LEVEL.WARNING:
-      icon = (
-        <i id="verify-warning" className="fas fa-exclamation-triangle fa-2x" />
-      );
-      break;
-    default:
-      icon = <i id="verify-invalid" className="fas fa-times-circle fa-2x" />;
-  }
-  return (
-    <div
-      className={`d-flex justify-content-center align-items-center ${
-        css["verified-icon"]
-      }`}
-    >
-      {icon}
-    </div>
+export const getIdentityVerificationText = identityDetails => {
+  const dnsNames = identityDetails.map(({ dns }) =>
```

Comment:
not sorted anymore, intended ? (I dunno why it was sorted)

Reply:
Yep there wasn’t a need to sort it